lib/type.ml

```diff
@@ -52,6 +52,12 @@ All possible cases defined in Yojson:
 - `Variant of (string * json option): Variant (non-standard extension of JSON).
 	    Syntax: [<"Foo">] or [<"Bar":123>].
 *)
+
+type json = t
+(**
```

Comment:
We should deprecate the alias so we can get rid of it when we bump the major version.